rclcpp/include/rclcpp/utilities.hpp

```diff
@@ -282,6 +284,27 @@ sub_will_underflow(const T x, const T y)
   return (y > 0) && (x < (std::numeric_limits<T>::min() + y));
 }
 
+/// Join values in a container turned into strings by a given delimiter
```

Comment:
I would rather add this in `rcpputils`. We should also move some stuff here to that package.